### DISCUSSION TITLE
failure in installation process

### DISCUSSION THREAD

**Discussion by ahmed26399** [atrocore/atrocore#489] [Q&A] [answered]
I faced this problem while i was trying to install AtroDam with demo data.
Any ideas
![image](https://user-images.githubusercontent.com/116101010/196484806-9059c5ff-bf5d-44a6-bef3-13a57c6cf41c.png)

**Comment by rratsun** [comment, marked as answer]
It looks like you did not configure you apache as it should.

Please, read [web-server-preparation-apache](https://github.com/atrocore/docs/blob/master/atrocore/admin-guide/web-server-preparation-apache.md) documentation attentively. 

I found an old closed issue with similar problem [issue](https://github.com/atrocore/atrocore/issues/288) maybe it helps.
